ModelCatalogueCorePluginTestApp/grails-app/assets/javascripts/modelcatalogue/core/ui/bs/navigationRightActions.coffee

```diff
@@ -355,7 +355,7 @@ angular.module('mc.core.ui.bs.navigationRightActions', ['mc.util.ui.actions', 'm
       label: "Import MC"
```

Comment:
@passionfruit18 you are on the good track 👍  this is actually the place which needed to be changed, not the title in the prompt bellow